priv/repo/seeds.exs

```diff
@@ -18,6 +18,14 @@ alias Oli.Accounts
 alias Oli.Accounts.{User, Author}
 alias Oli.Repo
 alias Oli.Utils.DataGenerators.NameGenerator
+alias Oli.Host.HostIdentifier
+import Ecto.Query
+
+unless Repo.exists?(HostIdentifier) do
```

Comment:
I think this is going in the right direction, but we still need a check here to actually read the value and compare with `System.get_env("HOST")` to determine whether to drop all oban jobs or not.

This implementation as it stands will only drop the oban jobs if the record in the DB doesn't exist. It may (and will) exist in DB clones and therefore this will not execute when it needs to

Reply:
Thanks for pointing that out. I just sent a [commit](https://github.com/Simon-Initiative/oli-torus/pull/4774/commits/45a03eb071e012a764333ed903cad089c1bc7a56) to handle it.